k8s/clusters/common.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+export KUBERNETES_VERSION="v1.9.7"
+
+export KOPS_DOMAIN="mdn.mozit.cloud"
+export KOPS_CLUSTER="k8s.${KOPS_REGION}.${KOPS_DOMAIN}"
+
+export KOPS_MASTER_COUNT=3
+export KOPS_MASTER_SIZE=m4.large
+export KOPS_MASTER_VOLUME_SIZE_GB=250
+export KOPS_NETWORKING=calico
+export KOPS_NODE_COUNT=3
```

Comment:
@limed Perhaps we should go immediately with `KOPS_NODE_COUNT=10` to match the current MozMEAO cluster, or would you prefer to increase it later when I spin-up the production instance of MDN?

Reply:
We can set it 10 but mozmeao has other apps running in the cluster does MDN require all 10? I'm fine with whatever the number here but just reminder of the cost of 10x m4.large. Also I'm making an assumption that stage and prod will be on the same cluster and separated  based on kubernetes namespaces, am I right to assume this?

Reply:
I agree, let's stick with 3 for now. I think adding a new node is easy, and staging is less than 10 pods.

I'd like to work w/ @jgmize or @metadave to figure out the load in our current cluster. We're probably not using the capabilities of 10 nodes at the moment, but with reserved instances we haven't been under as much pressure to reduce it.=

Reply:
The MozMEAO cluster that MDN currently uses does not host any other apps. It's only used for MDN stage and production. You're right that stage and prod are separated by Kubernetes namespaces.

Reply:
> The MozMEAO cluster that MDN currently uses does not host any other apps
I stand corrected :)

But yeah I'm ok with bumping up the number of nodes but for now lets keep it to 3

Reply:
Note that with https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler we can start with 3 and new nodes will be provisioned automatically as necessary.